src/software/world/robot_state.h

```diff
@@ -26,10 +27,11 @@ class RobotState
      * @param velocity The velocity of the robot, in metres per second
      * @param orientation The orientation of the robot
      * @param angular_velocity The angular velocity of the robot
```

Comment:
Not part of your changes, but I also feel like adding units for angular velocity would be helpful.

Reply:
I can't really because the `AngularVelocity` type abstracts away the exact unit. It's just a unit of `AngularVelocity`, not explicitly `radians` or `degrees`. The purpose of the `Angle` and `AngularVelocity` classes are to help avoid unit conversion issues like we might run into here if someone gave radians and the function expected degrees.